docs/documentation/actions/groovy.md

```diff
@@ -0,0 +1,46 @@
+This action executes a [Groovy](https://groovy-lang.org/documentation.html){:target="_blank"} script.
+
+=== "Inputs"
+
+    | Required | Name            | Type                     |     Default     |     Description                                |
+    |:--------:|:----------------|:-------------------------|:----------------|:-----------------------------------------------|
+    |    *     | `script`        | String                   |                 | The groovy script to be executed. The last statement must return a Map either implicitly or explicitly. </br> For a script that just launch background execution, an empty map ([:]) must be the last statement.    |
+    |          | `parameters`    | Map <String, Object>     |                 | Key/values parameters to be used in the script |
```

Comment:
We may add a note to highlight that a "logger" parameter is always present and contains the action logger ([DelegateLogger](https://github.com/chutney-testing/chutney/blob/master/engine/src/main/java/com/chutneytesting/engine/domain/execution/engine/parameterResolver/DelegateLogger.java))

Until now, it cannot be overridden